src/os/ui/areas.js

```diff
@@ -144,13 +147,25 @@ os.ui.AreasCtrl.prototype.exportDisabled = function() {
 };
 
 
+/**
+ * Default for the export button
+ *
+ * @export
+ */
+os.ui.AreasCtrl.prototype.defaultExport = function() {
+  os.ui.AreasCtrl.export({areaOpt: 'selected', scope: this.scope});
```

Comment:
The `areaOpt` and `scope` keys here either need to be referenced with quotes everywhere or without quotes everywhere. There's a reference below that uses them with quotes that won't work in compiled mode with this object.